app/screens/map/Map.js

```diff
@@ -259,7 +266,7 @@ export default class MapExample extends Component {
                       <HTML baseFontStyle={{ fontSize: 16, fontFamily: "Hoefler Text" }} html={post.title.rendered} />
                     </TouchableHighlight>
                     <Text style={{ fontSize: 12, fontFamily: "Helvetica-Bold", color: 'gray', paddingTop: 5 }}>
-                      {new Date(post.date).toLocaleDateString()}
+                      {new Date(post.date.replace(' ', 'T')).toLocaleDateString()}
```

Comment:
We don't need this anymore, now that the post date returns the regular format.
```suggestion
                      {new Date(post.date).toLocaleDateString()}
```